src/utils/GraphBuidler.cpp

```diff
@@ -0,0 +1,16 @@
+#include "GraphBuilder.h"
+#include "Position.h"
+#include "Time.h"
+#include "../bakery/Client.h"
+
+using namespace std;
+
+void defaultGraphBuilder(Graph &G){
+    Position *p1 = new Position(1, 1);
+    Position *p2 = new Position(2, 2);
+    Position *p3 = new Position(3, 3);
+
+    G.addVertex(*p1);
+    G.addVertex(*p2);
+    G.addVertex(*p3);
```

Comment:
We don't need to to allocate memory dynamically here. We can should do this and not worry about freeing it later
```suggestion
void defaultGraphBuilder(Graph &G){
    G.addVertex(Position(1, 1));
    G.addVertex(Position(2, 2));
    G.addVertex(Position(3, 3));
```
`Java messing with our minds 🙃`

Reply:
Oh forgot to change that. It was actually me trying to solve the problem with templates that sucked my life force. I'll fix it